distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -915,11 +923,7 @@ type Vector a
             True -> self.map it-> Comparable.from it
             False -> self.map it-> Comparable.from (on it)
         compare_funcs = comps.map (it-> it.compare)
-        problem_behavior = case on_incomparable of
```

Comment:
^^^ I cannot comment above due to lack of changes. The changes to `Array` Example with backticks added to `1` and `My_Type` should be ported to the Vector docstring as well.